<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTestWidgetClass</class>
 <widget class="QWidget" name="QTestWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTestWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QStaticTestWidget" name="tabStaticTest">
      <attribute name="title">
       <string>静态</string>
      </attribute>
     </widget>
     <widget class="QDynamicTestWidget" name="tabDynamicTest">
      <attribute name="title">
       <string>动态</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QInstTestWidget" name="tabInstTest">
      <attribute name="title">
       <string>瞬态</string>
      </attribute>
     </widget>
     <widget class="QCalibrationWidget" name="tabCalibration">
      <attribute name="title">
       <string>校准</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDynamicTestWidget</class>
   <extends>QWidget</extends>
   <header>qdynamictestwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QStaticTestWidget</class>
   <extends>QWidget</extends>
   <header>qstatictestwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCalibrationWidget</class>
   <extends>QWidget</extends>
   <header>qcalibrationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QInstTestWidget</class>
   <extends>QWidget</extends>
   <header>qinsttestwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
